app/src/main/res/values/strings.xml

```diff
@@ -1288,7 +1283,11 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_parkingType_underground">Underground garage</string>
     <string name="quest_parkingType_multiStorage">Parking garage</string>
 
-    <string name="quest_pedestrian_crossing_island">"Does this pedestrian crossing have an island?"</string>
+    <string name="quest_pedestrian_crossing_signals">"Does this crossing have traffic signals?"</string>
+
+    <string name="quest_pedestrian_crossing_island">"Does this crossing have an island?"</string>
```

Comment:
In case someone doesn’t understand the meaning of “island” in this context, how about something more descriptive like:

> Can a pedestrian cross halfway and wait at an island for the next walk signal?

Reply:
Sounds good, but also so complicated that I think it might be bordering on actually being confusing (which happens if you circumscribe a common thing instead of naming it). I've asked in the US #language chat and it seems to me that at least when the context is given (a crossing), "Island" is generally understood)

Reply:
New wording is confusing to me: what is wait? Wait for what? etc.